src/libraries/System.Linq.Queryable/src/System/Linq/Queryable.cs

```diff
@@ -11,8 +11,10 @@ namespace System.Linq
     public static class Queryable
     {
         internal const string InMemoryQueryableExtensionMethodsRequiresUnreferencedCode = "Enumerating in-memory collections as IQueryable can require unreferenced code because expressions referencing IQueryable extension methods can get rebound to IEnumerable extension methods. The IEnumerable extension methods could be trimmed causing the application to fail at runtime.";
+        internal const string InMemoryQueryableExtensionMethodsRequiresDynamicCode = "Enumerating collections as IQueryable can require creating new generic types or methods, which requires creating code at runtime. This may not work when AOT compiling.";
```

Comment:
Any options that we can recommend to the user to use instead of?

Reply:
I don't know of any, and there aren't any suggestions for trimming, so I don't think we'll add any here.